src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/service/JJDisputeService.java

```diff
@@ -175,10 +176,40 @@ public JJDispute updateJJDispute(String ticketNumber, JJDispute jjDispute, Princ
 				}
 			}
 		}
+		
+		// Update the LatestPleaUpdateTs if the LatestPlea is not the same as current one
+		for (JJDisputedCount jjDisputedCountToUpdate : jjDisputeToUpdate.getJjDisputedCounts()) {
+			JJDisputedCount jjDisputedCount = getJjDisputedCount(jjDispute, jjDisputedCountToUpdate.getCount());
+			if (jjDisputedCount != null) {				
+				Plea latestPlea = jjDisputedCount.getLatestPlea();
+				// If the new LatestPlea is null or does not match the current database value, set the LatestPleaUpdateTs to now
+				if (latestPlea == null || latestPlea.equals(jjDisputedCountToUpdate.getLatestPlea())) {
+					jjDisputedCountToUpdate.setLatestPleaUpdateTs(new Date());
+				}
```

Comment:
I think this logic needs to be updated a bit because the value of the `jjDisputedCountToUpdate.getLatestPlea()` is not coming from the database anymore since it has been replaced with the updated one from the request above at line 148. So we might need to get the jjDisputeToUpdate from the database again, or move this before that line to compare latestPlea database value.

Also, it currently checks if the values are equal before setting the `latestPleaUpdateTs` but it should check if it's not equal I think.